funnel/static/js/scripts.js

```diff
@@ -39,7 +39,7 @@ window.Hasgeek.Config = {
 
 function activate_widgets() {
   // Activate codemirror on all textareas with class='markdown'
-  $('textarea.markdown').each(function () {
+  $('textarea.markdown:not([style*="display: none"]').each(function () {
```

Comment:
activate widget fn is called again on new collaborator form in the modal. This is to avoid reinitialising codemirror enabled textarea (description field in the form)